code/modules/mob/living/carbon/human/human_defense.dm

```diff
@@ -196,6 +196,8 @@ emp_act
 	return 0
 
 /mob/living/carbon/human/proc/check_reflect(def_zone) //Reflection checks for anything in your l_hand, r_hand, or wear_suit based on the reflection chance var of the object
+	if(head.IsReflect(def_zone) == 1)
+		return 1
```

Comment:
this runtimes if the user isn't wearing a helmet. 
```suggestion
/mob/living/carbon/human/proc/check_reflect(def_zone) //Reflection checks for anything in your l_hand, r_hand, or wear_suit based on the reflection chance var of the object
	if(head.?.IsReflect(def_zone))
		return 1
```
and you don't need to do `== 1`

Reply:
should be solved now